editor/components/inserter/index.js

```diff
@@ -8,36 +8,55 @@ class Inserter extends wp.element.Component {
 	constructor() {
 		super( ...arguments );
 		this.toggle = this.toggle.bind( this );
-		this.close = this.close.bind( this );
+		this.update = true;
```

Comment:
Added to determine if we should run an update on the component (initial state is true).